adapter/config/ingress/status.go

```diff
@@ -68,14 +71,21 @@ func NewStatusSyncer(mesh *proxyconfig.ProxyMeshConfig, client kubernetes.Interf
 	if mesh.IngressService != "" {
 		publishService = fmt.Sprintf("%v/%v", options.Namespace, mesh.IngressService)
 	}
+	glog.Infof("INGRESS STATUS publishService ", publishService)
 	ingressClass, defaultIngressClass := convertIngressControllerMode(mesh.IngressControllerMode, mesh.IngressClass)
+
+	customIngressStatus := func(*betaext.Ingress) []v1.LoadBalancerIngress {
+		return nil
+	}
+
 	sync := status.NewStatusSyncer(status.Config{
 		Client:              client,
 		IngressLister:       store.IngressLister{Store: informer.GetStore()},
 		ElectionID:          ingressElectionID, // TODO: configurable?
 		PublishService:      publishService,
 		DefaultIngressClass: defaultIngressClass,
 		IngressClass:        ingressClass,
+		CustomIngressStatus: customIngressStatus,
```

Comment:
This works around a bug in k8s where there is no null check if that field is null and it causes the watcher to crash.